packages/jest-jasmine2/vendor/jasmine-2.4.1.js

```diff
@@ -703,7 +703,7 @@ getJasmineRequireObj().Env = function(j$) {
     var topSuite = new j$.Suite({
       env: this,
       id: getNextSuiteId(),
-      description: 'Jasmine__TopLevel__Suite',
+      description: 'test',
```

Comment:
renaming this so top level snapshot tests in Jest don't have a stupid name like "Jasmine__TopLevel__Suite".